packaging/windows/buildpackage.sh

```diff
@@ -63,22 +74,12 @@ function build_platform()
 	make install-dependencies "${TARGETPLATFORM}" gameinstalldir="" DESTDIR="${BUILTDIR}"
 	popd > /dev/null || exit 1
 
-	echo "Compiling Windows launchers ($1)"
-	makelauncher "${BUILTDIR}/RedAlert.exe" "Red Alert" "ra" RedAlert.ico $1
-	makelauncher "${BUILTDIR}/TiberianDawn.exe" "Tiberian Dawn" "cnc" TiberianDawn.ico $1
-	makelauncher "${BUILTDIR}/Dune2000.exe" "Dune 2000" "d2k" Dune2000.ico $1
-
-	# Windows specific files
-	cp OpenRA.ico RedAlert.ico TiberianDawn.ico Dune2000.ico "${BUILTDIR}"
+	echo "Compiling Windows launchers (${PLATFORM})"
+	makelauncher "RedAlert.exe" "Red Alert" "ra" ${PLATFORM}
+	makelauncher "TiberianDawn.exe" "Tiberian Dawn" "cnc" ${PLATFORM}
+	makelauncher "Dune2000.exe" "Dune 2000" "d2k" ${PLATFORM}
 	cp "${SRCDIR}/OpenRA.Game.exe.config" "${BUILTDIR}"
 
-	curl -s -L -O https://raw.githubusercontent.com/wiki/OpenRA/OpenRA/Changelog.md
```

Comment:
We don't package the changelog on the other platforms, and we routinely forget to finalize the changelog before pushing a tag causing outdated pages to be packaged. It is not clear that this gives a practical benefit to justify the foot-gun.